consensus/src/dag/dag_driver.rs

```diff
@@ -93,7 +97,7 @@ impl DagDriver {
             payload_client,
             reliable_broadcast,
             time_service,
-            rb_abort_handle: Mutex::new(None),
+            rb_handles: Mutex::new(BoundedVecDeque::new(window_size_config as usize)),
```

Comment:
Do we have to use the same window size here?

Reply:
We could increase, but let's keep this for now.